OneCommunityGlobal/HGNRest
# Shereen_Created Routing and controller function for Add Tool

Created Routing and controller function for Add Tool request.

# Description
Created Routing and controller function for Add Tool request.
Implements # (WBS) : 4.5.8

## Related PRS (if any):
This frontend PR is related to the #XXX backend PR.
To test this backend PR you need to checkout the[ #2306](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2306) frontend PR.


## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Follow the steps in frontend PR https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2306
